upgrade.md

```diff
@@ -33,6 +35,33 @@
 
 > {note} We attempt to document every possible breaking change. Since some of these breaking changes are in obscure parts of the framework only a portion of these changes may actually affect your application.
 
+<a name="php-8.0.2-required"></a>
+### PHP 8.0.2 Required
+
+**Likelihood Of Impact: Medium**
+
+The new minimum PHP version is now 8.0.2.
+
+<a name="updating-dependencies"></a>
+### Updating Dependencies
+
+Update the following dependencies in your `composer.json` file:
+
+<div class="content-list" markdown="1">
+- `laravel/framework` to `^9.0`
+- `nunomaduro/collision` to `^6.0`
+</div>
+
+In addition, replace `facade/ignition` by `"spatie/laravel-ignition": "^1.0"` in your `composer.json` file.
+
+The following first-party packages have new major releases to support Laravel 9.x. If applicable, you should read their individual upgrade guides before upgrading:
+
+<div class="content-list" markdown="1">
+    TODO...
```

Comment:
@driesvints Can you help me on this section please?

Reply:
@nunomaduro Only nexmo notification channel needs a new major release. I'm still working on that one.